mailpoet/lib/Segments/DynamicSegments/FilterDataMapper.php

```diff
@@ -323,6 +324,15 @@ private function createWooCommerce(array $data): DynamicSegmentFilterData {
       $filterData['operator'] = $data['operator'];
       $filterData['payment_methods'] = $data['payment_methods'];
       $filterData['used_payment_method_days'] = intval($data['used_payment_method_days']);
+    } elseif (in_array($data['action'], WooCommerceCustomerTextField::ACTIONS)) {
+      if (empty($data['value'])) throw new InvalidFilterException('Missing value', InvalidFilterException::MISSING_VALUE);
+      if (empty($data['operator'])) throw new InvalidFilterException('Missing operator', InvalidFilterException::MISSING_VALUE);
```

Comment:
I guess the exception here should be `InvalidFilterException::MISSING_OPERATOR` instead of `InvalidFilterException::MISSING_VALUE`?